autoload/airline/extensions/tabline.vim

```diff
@@ -12,7 +12,7 @@ endif
 
 
 function! airline#extensions#tabline#init(ext)
-  if has('gui_running')
+  if has('nvim') || has('gui_running')
```

Comment:
This change isn't needed, guioptions aren't used by neovim, not even by gui clients at the moment I think (and even in that case the original condition would be the right one)